CONTRIBUTING.md

```diff
@@ -97,20 +134,25 @@ when isMainModule:
 
 ```
 
-#### Submissions Requirements
+### Submissions Requirements
 
-- The file extension for algorithm files should be `.nim`.
-- Look up if the name of your algorithm already exists in another repository of [TheAlgorithms](https://github.com/TheAlgorithms/) like [TheAlgorithms/Python](https://github.com/TheAlgorithms/Python). By reusing the same name, your implementation will be regrouped with implementations of other languages.
-- Strictly use snake_case (underscore_separated) in your file_name, as it will be easy to parse in future using scripts.
-- Please open an issue first if you want to create a new directory. Try to fit your work into the existing directory structure as much as possible.
-- If possible, follow the standard *within* the folder you are submitting to.
-- If you have modified/added code work, make sure the code compiles before submitting. Please precise the version of the compiler you used in your Pull Request.
-- If you have modified/added documentation work, ensure your language is concise and contains no grammar errors.
+- Make sure the code compiles before submitting.
+- Look up the name of your algorithm in other active repositories of [TheAlgorithms](https://github.com/TheAlgorithms/), like [TheAlgorithms/Python](https://github.com/TheAlgorithms/Python). By reusing the same name, your implementation will be appropriately grouped alongside other implementations on the [project's web site](https://the-algorithms.com/).
+- Please help us keep our issue list small by adding fixes: Add the number of the issue you solved — even if only partially — to the commit message of your pull request.
+- Use *snake_case* (words separated with an underscore `_`) for the filename.
+- Try to fit your work into the existing directory structure as much as possible. Please open an issue first if you want to create a new subdirectory.
+- Writing documentation, be concise and check your spelling and grammar.
 - Add a corresponding explanation to [Algorithms-Explanation](https://github.com/TheAlgorithms/Algorithms-Explanation) (Optional but recommended).
+- Most importantly
+  * **Be consistent in the use of these guidelines when submitting.**
+  * **Join** us on [Discord](https://the-algorithms.com/discord) **now!**
```

Comment:
I am not sure to check regularly discord, but I think it is not bad to advertise for the algorithms organization.
The now is a bit too emphatic. Why would someone join the Discord?

Reply:
Well, I'm not on Discord and just left this line as it was. The proper place for this call is in the main Readme.

Reply:
I've deleted it. We already mention Discord in the header.